lib/tracer/trace.go

```diff
@@ -20,6 +20,10 @@ import (
 	"go.opentelemetry.io/otel/trace/noop"
 )
 
+const (
+	sampleRatio = 0.5 // Sample 50% of trace IDs
```

Comment:
With Grafana Tempo we can easily trace 100% since they use S3 compatible storage (significantly cheaper than solutions which use a DB).